src/climate_assessment/__init__.py

```diff
@@ -1,27 +1,33 @@
-from pathlib import Path
+"""
+Climate assesssment workflow
+"""
 
 import pyam
-from setuptools_scm import get_version
 
 try:
```

Comment:
I think this block might be outdated. As we've now restricted python to 3.11 importlib.metadata must be available

Reply:
Nice thanks